open-metadata-implementation/admin-services/admin-services-client/src/main/java/org/odpi/openmetadata/adminservices/client/MetadataAccessPointConfigurationClient.java

```diff
@@ -71,13 +71,20 @@ public MetadataAccessPointConfigurationClient(String adminUserId,
      */
 
     /**
-     * Return the list of access services for this server.
+     * Return the list of access services for this plaform
+     *
+     * This call is deprecated, as the MetadataAccessPointConfigurationClient client is for configuring a particular
+     * server, not interrogating platform information.
+     *
+     * Please use the call getRegisteredAccessServices in the platform configuration client
+     * OMAGServerPlatformConfigurationClient.
      *
      * @return list of access service descriptions
      * @throws OMAGNotAuthorizedException the supplied userId is not authorized to issue this command.
      * @throws OMAGInvalidParameterException invalid parameter.
      * @throws OMAGConfigurationErrorException unusual state in the admin server.
```

Comment:
This call is needed - do not deprecate

Reply:
OK - I am unsure what to put as the serverName in the constructor of this class. As the call is a platform orientated call without an associated server.
It ends up calling `AdminServicesRESTClient(String serverName,
                            String serverPlatformURLRoot) throws OMAGInvalidParameterException
    {
        final String  methodName = "RESTClient(no authentication)";

        this.serverPlatformURLRoot = serverPlatformURLRoot;

        RESTClientFactory factory = new RESTClientFactory(serverName, serverPlatformURLRoot);

        try
        {
            this.clientConnector = factory.getClientConnector();
        }
        catch (Exception error)
        {
            throw new OMAGInvalidParameterException(OMAGAdminErrorCode.NULL_LOCAL_SERVER_NAME.getMessageDefinition(serverName, error.getMessage()),
                                                    this.getClass().getName(),
                                                    methodName,
                                                    error);
        }
    }`
 The Exception and call seems to want a valid serverName.  Am I missing something?

Reply:
It is behaving correctly for the usecase it is designed for - ie configuring a server.  In this case the server name is the server being configured.  Requesting the registered access services is just one of the sequence of calls used to create a metadata access point and/or metadata server.

Use the platform services client if not configuring a specific server.

Reply:
ok will do

Reply:
done